RecoTracker/MkFitCore/src/MkFinder.h

```diff
@@ -0,0 +1,351 @@
+#ifndef RecoTracker_MkFitCore_src_MkFinder_h
+#define RecoTracker_MkFitCore_src_MkFinder_h
+
+#include "MkBase.h"
+#include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
+#include "RecoTracker/MkFitCore/interface/Track.h"
+
+#include "RecoTracker/MkFitCore/interface/HitStructures.h"
+
+// Define to get printouts about track and hit chi2.
+// See also MkBuilder::backwardFit().
+
+//#define DEBUG_BACKWARD_FIT_BH
+//#define DEBUG_BACKWARD_FIT
+
+namespace mkfit {
+
+  class CandCloner;
+  class CombCandidate;
+  class LayerOfHits;
+  class FindingFoos;
+  class IterationParams;
+  class IterationLayerConfig;
+  class SteeringParams;
+
+  //#ifdef DEBUG_BACKWARD_FIT
+  //class Event;
+  //#endif
+
+#ifdef DUMPHITWINDOW
+  class Event;
+#endif
+
+  // NOTES from MkFitter ... where things were getting super messy.
```

Comment:
Could this part be removed or moved?

Reply:
Addressed in Round 4 follow-up PR trackreco/cmssw#79 commit: https://github.com/trackreco/cmssw/pull/79/commits/85c149fb2b8af53d752e1a334f4d90feed9ba2ef